gpMgmt/bin/pythonSrc/PyGreSQL-4.0/pgmodule.c

```diff
@@ -1845,6 +1845,9 @@ pgquery_listfields(pgqueryobject * self, PyObject * args)
 	n = PQnfields(self->last_result);
 	fieldstuple = PyTuple_New(n);
 
+	if (fieldstuple == NULL)
```

Comment:
Maybe do it the same way as
https://github.com/arenadata/gpdb/blob/3b5aa3e9d5141832d94172e20e41a75770c07f34/gpMgmt/bin/pythonSrc/PyGreSQL-4.0/pgmodule.c#L876

Reply:
I think this approach is not very welcome.

Therefore I did as https://github.com/arenadata/gpdb/blob/3b5aa3e9d5141832d94172e20e41a75770c07f34/gpMgmt/bin/pythonSrc/PyGreSQL-4.0/pgmodule.c#L837-L839